sbnanalysis/ana/SBNOsc/config/NumuConfig.fcl

```diff
@@ -1,49 +1,172 @@
-OutputFile: "output_SBNOsc_NumuSelection.root"
+BEGIN_PROLOG
+// standard configurations to be used in proposal sensitivity
 
-NumuSelection: {
-  verbose: false
-  doFVCut: true
+eventweight_tags: ["fluxeventweight", "genieeventweight"]
 
-  // Using Uboone UBXSec analysis volume
-  fiducial_volumes: [
-    // first FV is the upstream one (relative to beam direction)
+SBND_fiducial_volumes: [
     {
-      xmin: 10.45
-      ymin: -80.53
-      zmin: 25.1
-      xmax: 242.8
-      ymax: 82.47
-      zmax: 675.1
+      xmin: -190.9
+      ymin: -185
+      zmin: 15.
+      xmax: 5.6
+      ymax: 185
+      zmax: 415.
     },
     {
-      xmin: 10.45
-      ymin: -80.53
-      zmin: 775.1
-      xmax: 242.8
-      ymax: 82.47
-      zmax: 951.9
+      xmin: 10.9
+      ymin: -185
+      zmin: 15.
+      xmax: 190.9
+      ymax: 185
+      zmax: 415.
     }
-  ]
+]
 
-  active_volumes: [
+SBND_active_volumes: [
     {
-     xmin: -1.55
-     ymin: -115.53
-     zmin: 0.1
-     xmax: 254.8
-     ymax: 117.47
-     zmax: 1036.9
+      xmin: -199.15
+      ymin: -200.
+      zmin: 0.0
+      xmax: 199.15
+      ymax: 200.
+      zmax: 500.
     }
-  ]
-
-  minLengthContainedTrack: 50, // value taken from proposal
-  minLengthExitingTrack: 100, // value taken from proposal
-  trackVisibleEnergyThreshold: 0.021, // [GeV] taken from pospoal
-  
-  showerEnergyDistortion: 0.05, // value taken from tech note by J. Zennamo and C. Adams
-  trackEnergyDistortion: 0.05, // same as above
-  leptonEnergyDistortionContained: 0.02, // same as above
-  leptonEnergyDistortionLeavingA: 0.0, // dp/p = -Alog(BL) in tech note. Still needs to be set
-  leptonEnergyDistortionLeavingB: 0.0, // same as above
+]
+
+// fiducial volume of SBND used in proposal
+SBND_Proposal_fiducial_volumes: [
```

Comment:
Might be cleaner to make this hierarchical, and reference e.g. `sbnd.volumes.proposal.fiducial` or similar? (Also, should the experiment-specific things like this be in the config that everything includes?)

Reply:
Perhaps -- this configuration was small enough that I thought it was simpler to keep it in one file. Also, this definition of the fiducial/active volumes is very particular to the NuMu analysis and I would not expect other selections to use it (even Nue, which had a different volume definition in the Proposal).